core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -388,6 +388,16 @@ public void setStateTest() throws Exception {
   }
 
   /**
+   * Tests the permission bits are 0755 for directories and 0644 for files by default.
+   */
+  @Test
+  public void permissionTest() throws Exception {
+    mFileSystemMaster.createFile(NESTED_FILE_URI, sNestedFileOptions);
+    Assert.assertEquals(0755, mFileSystemMaster.getFileInfo(NESTED_URI).getPermission());
```

Comment:
each of these two permissions appeared once here. it will be a bit strange if they are defined here but only get used once.